test/test_transaction.rb

```diff
@@ -57,64 +57,60 @@ def test_basic_transaction_setters
   end
 
   def test_test_flag_install
-    skip("hanging")
-
-    filename = 'simple-1.0-0.i586.rpm'
-    pkg = RPM::Package.open(fixture(filename))
+    pkg = RPM::Package.open(fixture(PACKAGE_FILENAME))
```

Comment:
The PACKAGE_FILENAME constant is the same as `filename` so we might as well use the constant